app/grandchallenge/hanging_protocols/templates/hanging_protocols/hangingprotocol_detail.html

```diff
@@ -18,7 +18,10 @@
 {% endblock %}
 
 {% block content %}
-    <h3>{{ object.title }}</h3>
+    <h3>
+        <span style="stroke: #000">{{ object.svg_icon|safe }}</span>

```

Comment:
Best to avoid using `safe` in templates. Same for `app/grandchallenge/hanging_protocols/templates/hanging_protocols/hangingprotocol_list.html`.